### PR TITLE
Default page

### LINKED ISSUE #7: Display Default Page
Update reminder component to have a conditional that will only display notes if notes exist.

There should be a default welcome page welcoming users to the app and/or directions to create a new note. Feel free to be creative here.

### PR DESCRIPTION
**Pull request by maiastone**
## Purpose

This pull request achieves the following:   
Updates reminder to have a conditional that will only display notes if notes exist.   
Adds a default welcome page welcoming users to the app and/or directions to create a new note.
This Closes #7 

## Approach

Adding the conditional was straightforward.
The default page was created using the index.hbs template, and the transitionTo was removed from the index route to allow the default page to be viewed at the root URL.  A link-to helper was added to the welcome page, allowing the user to click to the reminders page which shows the list of reminders.   
The Ember-Inline-SVG add-on was used to render the logo on the welcome page.   

### Learning

[Ember Docs on Conditionals](https://guides.emberjs.com/v2.8.0/templates/conditionals/)   
[Ember Docs on Link-to Helpers](https://guides.emberjs.com/v2.8.0/templates/links/)   
[Ember-Inline-SVG](https://github.com/minutebase/ember-inline-svg)   
 
### Test coverage 

An acceptance test was added to reflect the new routing of the default page.   

### Follow-up tasks

- [Issue #8 ](https://github.com/turingschool-projects/1608-remember-5/issues/8)

### Screenshots

@nfosterky @martensonbj @Tman22

### CODE REVIEW

**Comment by maiastone** [comment]
@martensonbj @nfosterky @Tman22 
This branch now has no merge conflicts.  Please review PR 🎉

File: app/routes/index.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-  beforeModel() {
-    this.transitionTo('reminders');
-  }
```
Comment (Tman22) [inline app/routes/index.js]: You should keep this.

File: app/templates/index.hbs [hunk 2]
```diff
@@ -1 +1,16 @@
+<section>
```
Comment (Tman22) [inline app/templates/index.hbs]: This should be in the the reminders.hbs, then reminders should be switched to reminders/index.hbs

File: tests/acceptance/reminder-list-test.js [hunk 3]
```diff
@@ -7,36 +7,43 @@ import Ember from 'ember';
 
 moduleForAcceptance('Acceptance | reminders list');
 
-test('viewing the homepage', function(assert) {
+test('viewing the reminders page', function(assert) {
   server.createList('reminder', 5);
-  visit('/');
```
Comment (Tman22) [inline tests/acceptance/reminder-list-test.js]: You can keep this once you change the reminders back.

File: tests/acceptance/reminder-list-test.js [hunk 4]
```diff
@@ -7,36 +7,43 @@ import Ember from 'ember';
...
     assert.equal(currentURL(), '/reminders');
     assert.equal(Ember.$('.spec-reminder-item').length, 5);
   });
 });
 
-test('viewing the homepage', function(assert) {
+test('viewing the reminders', function(assert) {
   server.createList('reminder', 10);
-  visit('/');
+  visit('/reminders');
```
Comment (Tman22) [inline tests/acceptance/reminder-list-test.js]: See above ^^

File: tests/acceptance/reminder-list-test.js [hunk 5]
```diff
@@ -7,36 +7,43 @@ import Ember from 'ember';
...
-
   andThen(function() {
     assert.equal(currentURL(), '/reminders/1');
     assert.equal(Ember.$('.spec-reminder-item:first').text().trim(), Ember.$('.spec-reminder-title').text().trim());
   });
 });
 
 test('Clicking the new reminder button will redirect the user to the route "/new"', function(assert) {
-  visit('/');
+  visit('/reminders');
```
Comment (Tman22) [inline tests/acceptance/reminder-list-test.js]: see above ^

**Comment by maiastone** [comment]
Please review the default page-- should be good and Closes #7 🐻  @Tman22

File: app/templates/index.hbs (hunk 2 above)
Comment (martensonbj) [inline app/templates/index.hbs]: Just need to delete this file now that you've moved it to the reminders/index.hbs

File: tests/acceptance/reminder-list-test.js (hunk 5 above)
Comment (martensonbj) [inline tests/acceptance/reminder-list-test.js]: This should still be `visit('/')` - it should always route you to `/reminders` so there's never a need to explicitly visit that route.

**Comment by maiastone** [comment]
🎉  😄   Alrighty, then.  @martensonbj @Tman22 @nfosterky

**Pull request merged**